lib/wallaby/browser.ex

```diff
@@ -1012,6 +1012,26 @@ defmodule Wallaby.Browser do
     )
   end
 
+  @doc """
+  Finds the shadow DOM for the specified element and returns an element corresponding
+  to this shadow DOM. Subsequent queries made within element retured by calling `shadow_root`
+  will return elements within the shadow DOM, e. g.
+
+  ```
+  element =
+    session
+    |> find(Query.css("shadow-test"))
+    |> shadow_root()
+    |> find(Query.css("#in-shadow"))
+  ```
```

Comment:
```suggestion
  ```elixir
  session
  |> find(Query.css("shadow-test"))
  |> shadow_root()
  |> find(Query.css("#in-shadow"))
  ```
```